presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -38,12 +38,20 @@ Constructors
 
     Returns a geometry type object from WKT representation.
 
+.. function:: ST_GeometryFromBinary(varbinary) -> Geometry
```

Comment:
Functions here are listed in alphabetical order. Could you move these new functions accordingly?

Also, the function name in the SQL/MM Part 3 specification is `ST_GeomFromWKB`, not `ST_GeometryFromWKB`. Could you please update? (I don't know why we have `ST_GeometryFromText` and not `ST_GeomFromText`. This needs to change as well at some point.)

<img width="805" alt="screen shot 2018-09-04 at 10 43 32 am" src="https://user-images.githubusercontent.com/27965151/45038587-83716b80-b02f-11e8-8179-c8b90a2b0bd1.png">

Reply:
@mbasmanova The functions in this section are not in alphabetical order, so I can not put the new functions in that way. Should I reorder the functions in this section or would that have to be in a separate PR?

Reply:
@mpertierra Oh... Thanks for pointing it out. Would you fix the order of existing functions in a separate commit? All commits can be part of this PR. No need to create a new PR.